hummingbot/client/liquidity_bounty/bounty_utils.py

```diff
@@ -134,6 +142,55 @@ def format_volume_metrics(volume_metrics: List[Dict[str, Any]]) -> str:
         except Exception as e:
             self.logger().error(f"Failed to query for unsubmitted trades: {str(e)}", exc_info=True)
 
+    def get_order_filter(self) -> BooleanClauseList:
+        and_conditions: BooleanClauseList = [and_(
+            Order.base_asset == ab["base_asset"],
+            Order.market == ab["market"],
+            Order.creation_timestamp >= ab["start_timestamp"],  # does not matter if start_timestamp == -1
+            Order.creation_timestamp <= (ab["end_timestamp"] if ab["end_timestamp"] > 0 else 1e14)
+        ) for ab in self._active_bounties]
+        return and_conditions
+
+    async def get_unsubmitted_orders(self) -> List[Order]:
+        """ Get locally saved orders that have not been submitted to liquidity bounties """
+        await self._wait_till_ready()
+        session: Session = SQLConnectionManager.get_trade_fills_instance().get_shared_session()
+
+        try:
+            and_conditions: BooleanClauseList = self.get_order_filter()
+
+            query: Query = (session
+                            .query(Order)
+                            .filter(Order.creation_timestamp > self._last_submitted_order_timestamp)
+                            .filter(or_(*and_conditions))
+                            .order_by(Order.creation_timestamp))
+
+            new_orders: List[Order] = query.all()
+            return new_orders
+        except Exception as e:
+            self.logger().error(f"Failed to query for unsubmitted orders: {str(e)}", exc_info=True)
+
+    async def get_unsubmitted_order_statuses(self) -> List[OrderStatus]:
```

Comment:
We want to submit order status that are older than the last submitted order. 
For the query, add another filter so that all order status has a smaller ts than last submitted order timestamp.